engine/lib/cron.php

```diff
@@ -35,6 +35,58 @@ function _elgg_cron_init() {
 }
 
 /**
+ * Cron run
+ *
+ * This function was designed to be called every one minute from a cron job to
+ * executes each Elgg cron period at the desired interval.  Will also exeute
+ * any Elgg cron period that have not fired by the expected deadline.
+ *
+ * Can be called manually by: http://YOUR.SITE/cron/run/
+ *
+ * This will only execute cron periods at specified intervals to force execution
+ * of a specific period you will need to use http://YOUR.SITE/cron/<period>/
+ *
+ * @access private
+ */
+function _elgg_cron_run() {
+	$now = time();
+	$params = array();
+	$params['time'] = $now;
+
+	// Data to return to
+	$old_stdout = "";
+	$std_out = "";
+	ob_start();
+
+	$periods = array(
+		'minute' => 60,
+		'fiveminute' => 300,
+		'fifteenmin' => 900,
+		'halfhour' => 1800,
+		'hourly' => 3600,
+		'daily' => 86400,
+		'weekly'=> 604800,
```

Comment:
The code analysis tests are failing because there should be a space between ```'weekly'``` and ```=>```.